src/socket.cr

```diff
@@ -179,15 +195,15 @@ class Socket < IO
   # sock = Socket.udp(Socket::Family::INET)
   # sock.bind Socket::IPAddress.new("192.168.1.25", 80)
   # ```
-  def bind(addr)
+  private def bind(addr, addrstr)
```

Comment:
It seems we lost `sock.bind Socket::IPAddress.new("192.168.1.25", 80)` since the method is now private. A miss edition probably/